local-setup/README.md

```diff
@@ -0,0 +1,36 @@
+#  How to use the local-setup
+
+## Prerequisite
+- worker built with `make SGX_MODE=SW`
```

Comment:
Sorry to complain again, but usually env-vars are before the cmd: `SGX_MODE=SW make`

Reply:
ops. Thanks !

Reply:
Fixed